diff -ruN wetterdienst-0.73.0/pyproject.toml wetterdienst-0.73.0-patch/pyproject.toml
--- wetterdienst-0.73.0/pyproject.toml	2024-02-09 00:13:05.951434400 +0100
+++ wetterdienst-0.73.0-patch/pyproject.toml	2024-02-09 20:10:49.328776683 +0100
@@ -211,6 +211,7 @@
 bufr                = ["pybufrkit", "pdbufr"]
 cratedb             = ["pandas", "crate"]
 duckdb              = ["pandas", "duckdb"]
+eccodes             = ["eccodes"]
 explorer            = ["dash", "dash-bootstrap-components", "dash-leaflet", "geojson", "plotly"]
 export              = ["pandas", "openpyxl", "sqlalchemy", "xarray", "zarr"]
 influxdb            = ["influxdb", "influxdb-client", "influxdb3-python"]
